### PR TITLE
ACR FHIR Extension

### PR DESCRIPTION
**Pull request by mocsharp**
### Description
* Support of pulling FHIR resources.
* Extends the ACR specs to support retrieval of both DICOM & FHIR resources in a single call for a single job.
* Update `VERSION` file to 0.8.2

### Status
**Work in progress**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [x] New tests added to cover the changes.
- [x] All tests passed locally by running `./src/run-tests-in-docker.sh`.
- [x] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.
- [x] User guide updated.
- [ ] NGC publishing metadata updated.
- [x] I have updated the changelog

### CODE REVIEW

**Review by slbryson — COMMENTED** [review]
Great work adding the FHIR extensions. Looked through all the changes no comments.

**Comment by rahul-imaging** [comment]
Looks good

File: src/API/Rest/DicomWebConnectionDetails.cs [hunk 1]
```diff
@@ -25,6 +25,23 @@ namespace Nvidia.Clara.DicomAdapter.API.Rest
...
+        [JsonProperty("baseUri")]
+        private string BaseUri { set { Uri = value; } }
+
+        /// <summary>
+        /// Gets or sets ApiKey.
+        /// This property is intentionally private and maps to AuthId to reduce exposed properties.
+        /// </summary>
+        /// <value></value>
+        [JsonProperty("apiKey")]
+        private string ApiKey { set { AuthId = value; } }
```
Comment (whoisj) [inline src/API/Rest/DicomWebConnectionDetails.cs:43]: is the connection encrypted? best to not send/receive credentials when the connection isn't encrypted.

**Review by whoisj — APPROVED** [review]
Lgtm

File: src/API/Rest/DicomWebConnectionDetails.cs (hunk 1 above)
Comment (mocsharp) [reply src/API/Rest/DicomWebConnectionDetails.cs:43]: Good call..  I will put a note in the user guide.

**Pull request merged**